src/cephadm/cephadm.py

```diff
@@ -2031,6 +2031,11 @@ def get_hostname():
     return socket.gethostname()
 
 
+def get_short_hostname():
+    # type: () -> str
+    return socket.gethostname().split('.', 1)[0]
```

Comment:
May I suggest that you call get_hostname so that if the implementation of get_hostname is changed or Mocked out, no extra work needs to be done for get_short_hostname.